site/content/blog/2024-09-18-detachable-tabs/index.md

```diff
@@ -0,0 +1,27 @@
+---
+title: "ZAP now has Detachable Tabs!"
+summary: >
+  ZAP tabs can now be opened in new windows.
+images:
+- https://www.zaproxy.org/blog/2024-09-18-detachable-tabs/images/zap-detachable-tabs.png
+type: post
+tags:
+- blog
+- update
+- ui
+date: "2024-09-18"
+authors:
+- akshath
+---
+
+![An image containing screenshots of two ZAP windows. One is the ZAP workbench with a highlighted context menu item on the Request-Response tab with the text "Move Tab to New Window". The other window is the Request-Response panel."](images/zap-detachable-tabs.png)
+
+ZAP tabs can now be opened in new windows.
+This includes the tabs in the workspace, tree, and information panels.
+
+To pop out a tab into a new window, simply right click on it, and select "Move Tab to New Window".
```

Comment:
How can they pop it back in? (I know the answer, but other people wont).
Does this work in all of the ZAP screen modes?
Are there any gotchas?